src/app/app.module.ts

```diff
@@ -16,7 +16,6 @@ import { TodoService } from './todo.service';
   ],
   imports: [
     BrowserModule,FormsModule,RouterModule.forRoot([
-      {path:'',redirectTo:'todo',pathMatch:'full'},
```

Comment:
is your app working without path '' ??
fix this

Reply:
yes sir its working!